src/Discord/functions.php

```diff
@@ -181,8 +181,9 @@ function imageToBase64(string $filepath): string
     }
 
     $mimetype = \mime_content_type($filepath);
+	$allowed = ['image/jpeg', 'image/png', 'image/gif'];
```

Comment:
```suggestion
    $allowed = ['image/jpeg', 'image/png', 'image/gif'];
```
Use 4 spaces instead of 1 tab